client/src/App.tsx

```diff
@@ -27,7 +28,8 @@ function App(): JSX.Element {
                   <Route exact path="/login" component={Login} />
                   <Route exact path="/signup" component={Signup} />
                   <ProtectedRoute exact path="/listings" component={Listings} />
-                  <ProtectedRoute exact path="/user/profile" component={UserProfile} />
+                  <ProtectedRoute exact path="/user/editProfile" component={EditProfile} />
+                  <ProtectedRoute exact path="/user/profilePhoto" component={ProfilePhoto} />
```

Comment:
I'm not sure making entirely separate components is the best approach here as that would require you to repeat a lot of code meaning changes in the future would be more difficult. A different approach you could try is having one route for profile settings that renders the sidebar and card and have a path param that tells which component to render within that card.

Reply:
Excellent suggestion! I have implemented it and my code looks much cleaner.